components/script/dom/webgl_extensions/ext/oesstandardderivatives.rs

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use dom::bindings::codegen::Bindings::OESStandardDerivativesBinding;
+use dom::bindings::codegen::Bindings::OESStandardDerivativesBinding::OESStandardDerivativesConstants;
+use dom::bindings::js::Root;
+use dom::bindings::reflector::{DomObject, Reflector, reflect_dom_object};
+use dom::webglrenderingcontext::WebGLRenderingContext;
+use dom_struct::dom_struct;
+use super::{WebGLExtension, WebGLExtensions};
+
+#[dom_struct]
+pub struct OESStandardDerivatives {
+    reflector_: Reflector,
+}
+
+impl OESStandardDerivatives {
+    fn new_inherited() -> OESStandardDerivatives {
+        Self {
+            reflector_: Reflector::new(),
+        }
+    }
+}
+
+impl WebGLExtension for OESStandardDerivatives {
+    type Extension = OESStandardDerivatives;
+    fn new(ctx: &WebGLRenderingContext) -> Root<OESStandardDerivatives> {
+        reflect_dom_object(box OESStandardDerivatives::new_inherited(),
+                           &*ctx.global(),
+                           OESStandardDerivativesBinding::Wrap)
+    }
+
+    #[cfg(any(target_os = "android", target_os = "ios"))]
+    fn is_supported(ext: &WebGLExtensions) -> bool {
```

Comment:
maybe it's easier to instead of `#[cfg(..)]` the method definition, just to write inside:

```rust
if cfg!(any(target_os = "android", traget_os = "ios")) {
    return false;
}
ext.supports_any_gl_extension(..)
```

That would avoid some duplication.